react/SearchTitle.js

```diff
@@ -58,14 +58,21 @@ const SearchTitle = props => {
 
   const title =
     index >= 0 ? breadcrumbName(index, breadcrumb) : getLastName(breadcrumb)
+
+  const decodedTitle = useMemo(() => {
+    try {
+      return decodeURI(title)
+    } catch {
+      return title
+    }
+  }, [title])
+
   if (!title) {
```

Comment:
I think this check should be done before the `useMemo` hook. Calling `decodeURI` on `null` or `undefined` returns a string, so this makes this condition unreachable. It also deviates from the previous behavior.

Reply:
Ooops, my bad! I thought you were using decodedTitle 😅 My bad!